docs/hub/datasets-distilabel.md

```diff
@@ -0,0 +1,110 @@
+# Distilabel
+
+Distilabel is the framework for synthetic data and AI feedback for AI engineers that require high-quality outputs, full data ownership, and overall efficiency.
```

Comment:
I wonder if we could simplify this into a more direct, explicit messaging (as suggested for Argilla)